compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/PlatformInsets.uikit.kt

```diff
@@ -54,7 +55,7 @@ private object SafeAreaInsetsConfig : InsetsConfig {
         CompositionLocalProvider(
             LocalSafeArea provides if (safeInsets) PlatformInsets() else safeArea,
             LocalLayoutMargins provides if (safeInsets) layoutMargins.exclude(safeArea) else layoutMargins,
-            LocalKeyboardOverlapHeight provides if (ime) 0.dp else keyboardOverlapHeight,
+            LocalKeyboardOverlapHeight provides if (ime) 0.dp else max(0.dp, keyboardOverlapHeight - safeArea.bottom),
```

Comment:
Minor: `.coerceAtLeast(0.dp)` is slightly more ideomatic.

Reply:
Thanks, done